rapids-cmake/cpm/libcudacxx.cmake

```diff
@@ -54,6 +54,32 @@ function(rapids_cpm_libcudacxx)
   include("${rapids-cmake-dir}/cpm/detail/package_details.cmake")
   rapids_cpm_package_details(libcudacxx version repository tag shallow exclude)
 
+  set(to_install OFF)
+  if(INSTALL_EXPORT_SET IN_LIST ARGN AND NOT exclude)
+    set(to_install ON)
+    # By default if we allow libcudacxx to install into `CMAKE_INSTALL_INCLUDEDIR` alongside rmm (or
+    # other packages) we will get a install tree that looks like this:
+
+    # include/rmm include/cub include/libcudacxx
+
+    # This is a problem for CMake+NVCC due to the rules around import targets, and user/system
+    # includes. In this case both rmm and libcudacxx will specify an include path of
+    # `include`, while libcudacxx tries to mark it as an user include, since rmm uses
+    # CMake's default of system include. Compilers when provided the same include as both user and
+    # system always goes with system.
+
+    # Now while rmm could also mark `include` as system this just pushes the issue to
+    # another dependency which isn't built by RAPIDS and comes by and marks `include` as
+    # system.
```

Comment:
Fix style for CI:
```suggestion
    # includes. In this case both rmm and libcudacxx will specify an include path of `include`,
    # while libcudacxx tries to mark it as an user include, since rmm uses CMake's default of system
    # include. Compilers when provided the same include as both user and system always goes with
    # system.

    # Now while rmm could also mark `include` as system this just pushes the issue to another
    # dependency which isn't built by RAPIDS and comes by and marks `include` as system.
```